deploy-transact/lib/pantheon-node-signing.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+
+const EEAClient = require("web3-eea");
+const Web3 = require('web3');
+const axios = require('axios');
+const solc = require('solc');
+const join = require('path').join;
+const fs = require('fs-extra');
+const argv = require('yargs').argv;
+const verbose = argv.verbose;
+
+class PantheonNodeSigningHandler {
```

Comment:
suggest extending `NodeSignerHandler` to avoid duplicating code that are completely common, for instance `getContract()` and `getAccount()`